tests/virt_autotest/guest_upgrade_run.pm

```diff
@@ -44,6 +44,12 @@ sub get_script_run {
     my $guest_list = get_required_var("GUEST_LIST");
 
     $pre_test_cmd = "$pre_test_cmd -p $product_upgrade -r $product_upgrade_repo -g \"$guest_list\"";
+    my $do_registration = check_var('GUEST_SCC_REGISTER', 'installation') ? "true" : "false";
+    my $registration_server = get_var('SCC_URL', 'https://scc.suse.com');
+    my $registration_code = get_var('SCC_REGCODE', 'INVALID_REGCODE');
+    $pre_test_cmd .= " -e $do_registration";
+    $pre_test_cmd .= " -s $registration_server";
+    $pre_test_cmd .= " -c $registration_code";
```

Comment:
It seems we already have this
```
    my $do_registration = check_var('GUEST_SCC_REGISTER', 'installation') ? "true" : "false";
    my $registration_server = get_var('SCC_URL', 'https://scc.suse.com');
    my $registration_code = get_var('SCC_REGCODE', 'INVALID_REGCODE');
    $pre_test_cmd .= " -e $do_registration";
    $pre_test_cmd .= " -s $registration_server";
    $pre_test_cmd .= " -c $registration_code";
```
in master branch ???

Reply:
Refer to current  guest_upgrade_run.pm in master branch, during s390x guest upgrade test run, there will be called as 
 `/usr/share/qa/virtautolib/lib/test_full_guest_upgrade.sh -p sles-15-sp6-64 -r http://openqa.suse.de/assets/repo/SLE-15-SP6-Full-s390x-Build45.1-Media1 -g \"sles-15-sp5-64\"  | tee /tmp/s390x_guest_upgrade_test.log -e false -s http://all-45.1.proxy.scc.suse.de/ -c 25af9a3177447b4a 2>&1 `
 Yes, I have to move it upwards ... 
 **AS THE SAME**, The validation run for this code change without s390x arch.